detection-rules/homoglyph_sender_domain_template.yml

```diff
@@ -5,10 +5,10 @@ type: "rule"
 source: |
   type.inbound
   and (
-      iedit_distance(sender.email.domain.domain, 'YOURDOMAIN1.COM') < 2
-      or iedit_distance(sender.email.domain.domain, 'YOURDOMAIN2.COM') < 2
+      strings.ilevenshtein(sender.email.domain.domain, 'YOURDOMAIN1.COM') < 2
```

Comment:
could ignore this one, we're about to drop this template rule: https://github.com/sublime-security/sublime-rules/pull/177